@xen-orchestra/lite/src/locales/en.json

```diff
@@ -56,16 +53,27 @@
   "property": "Property",
   "ram-usage": "RAM usage",
   "reboot": "Reboot",
+  "relative-time": {
+    "day": "1 day | {n} days",
+    "future": "In {str}",
```

Comment:
I wonder if this could be hard to internationalize in some languages. Maybe some languages use "{n} word {unit}" (with the amount and the unit separated in the sentence), or maybe some languages use a different word depending on the unit and/or the amount. Just a thought, because I'm not sure we can fix that issue without making it over-complicated.

Reply:
I think this could indeed be a problem with some languages.

We'll adapt the code when the time comes, if ever.